### PR TITLE
Refactor classmonitor

### PR DESCRIPTION
**Pull request by laksherthanlife**
Lets change all cases of addressbook(inclusive of its various case styles) into classmonitor (with the matching case style respectively)

### CODE REVIEW

File: src/test/data/JsonSerializableClassMonitorTest/typicalStudentClassMonitor.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "_comment": "AddressBook save file which contains the same Student values as in TypicalStudents#getTypicalAddressBook()",
+  "_comment": "classMonitor save file which contains the same Student values as in TypicalStudents#getTypicalClassMonitor()",
```
Comment (evedaktyl) [inline src/test/data/JsonSerializableClassMonitorTest/typicalStudentClassMonitor.json]: ClassMonitor?

**Review by evedaktyl — COMMENTED** [review]
Lgtm! Thanks Laksh

File: docs/diagrams/ArchitectureSequenceDiagram.puml [hunk 2]
```diff
@@ -20,7 +20,7 @@ activate model MODEL_COLOR
 model -[MODEL_COLOR]-> logic
 deactivate model
 
-logic -[LOGIC_COLOR]> storage : saveAddressBook(addressBook)
+logic -[LOGIC_COLOR]> storage : saveAddressBook(classMonitor)
```
Comment (chiaryan) [inline docs/diagrams/ArchitectureSequenceDiagram.puml]: should be `saveAddressBook`?

File: docs/diagrams/StarCommandSequenceDiagram.puml [hunk 3]
```diff
@@ -4,7 +4,7 @@ skinparam ArrowFontStyle plain
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
-participant "addressBookParser\n:AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "classMonitorParser\n:AddressBookParser" as AddressBookParser LOGIC_COLOR
```
Comment (chiaryan) [inline docs/diagrams/StarCommandSequenceDiagram.puml]: should be `ClassMonitorParser LOGIC_COLOR`?

**Review by chiaryan — COMMENTED** [review]
Not many changes needed. If you want to be more thorough you could recompile the PUML diagrams also but I think this is good enough

**Pull request merged**